js/coinex.js

```diff
@@ -1822,6 +1823,31 @@ module.exports = class coinex extends Exchange {
         return this.parseTrades (trades, market, since, limit);
     }
 
+    async setLeverage (leverage, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' setLeverage() requires a symbol argument');
+        }
+        if (params['position_type'] === undefined) {
```

Comment:
This conditional is a violation of this rule: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys